### PR TITLE
feat: made ecom responsive

### PR DESCRIPTION
**Pull request by harshalini**
Made ecommerce responsive for mobile view, tablet view and desktop view.

preview -
https://625814433c52760b525a720d--adorable-alpaca-991471.netlify.app/

### CODE REVIEW

**Review by sushil-mod — APPROVED** [review]
UI looks nice. Good responsiveness and i liked how you changed on hover to on click for small screen for add to cart/go to cart button .

**Review by ashishnnnnn — APPROVED** [review]
Code Seems Pretty much clean..
Good Work on UI part :+1:

**Review by ajit-sharmaGH — APPROVED** [review]
Code looks clean
Good to merge 
UI is good 
Color combination is good 
Suggestion : in mobile view make the filter page be hidden initially and then should be toggled by clicking on hamburger !

File: src/styles/products.css [hunk 1]
```diff
@@ -145,24 +164,47 @@ button.addCart-btn {
...
         width: 10rem;
         height: 2.2rem;
         text-align: center;
-        margin-top: 5.5rem;
+        margin-top: 20rem;
         font-size: 1rem;
+        display: block;
+    }
+    .addCart-btn:hover {
+        background: linear-gradient(var(--white), var(--orange));
```
Comment (ajit-sharmaGH) [inline src/styles/products.css:184]: Good practice of using colors syntax

File: src/styles/products.css [hunk 2]
```diff
@@ -145,24 +164,47 @@ button.addCart-btn {
...
     }
+    #idealFor-men, #idealFor-women, #shoes-category {
+        margin-left: -4rem;
+    }
+    span[for="idealFor-men"], span[for="idealFor-women"], span[for="shoes-category"] {
+        margin-left: -7rem;
+    }
+    input.rating-input {
+        margin-left: -7rem;
+        margin-right: -6rem;
```
Comment (ajit-sharmaGH) [inline src/styles/products.css]: It can be reduced to a single line `margin:0  -6rem  0  -7rem`

**Review by ajit-sharmaGH — APPROVED** [review]

**Review by Omkar-Chaskar — APPROVED** [review]
Css and react file structure is good. 
Clean and understandable code.
UI is clean with on hover shadow effect and add-to-cart  button visibility. I actually like that one. All components are well coded and designed.
good to merge.

**Comment by harshalini** [comment]
> Code looks clean Good to merge UI is good Color combination is good Suggestion : in mobile view make the filter page be hidden initially and then should be toggled by clicking on hamburger !

Ok, thank you!

**Pull request merged**